pyseir/inference/infer_rt.py

```diff
@@ -190,12 +190,11 @@ def apply_gaussian_smoothing(self, timeseries_type, plot=False):
                              .mean(std=self.kernel_std)\
                              .round()
 
-        zeros = smoothed.index[smoothed.eq(0)]
-        if len(zeros) == 0:
+        nonzeros = [idx for idx, val in enumerate(smoothed) if val!= 0]
+        if len(nonzeros) == 0:
             idx_start = 0
```

Comment:
So if there are any zeros (the below case), we skip to the first non-zero.  If there are _only_ zeros (this case), we include them all?

Intuitively, I'd expect that we would skip them all instead...  but when you smooth zeros, you just get back zeros, so it doesn't matter?

Is it worth a comment?

> If we have a mix of zeros and nonzeros, skip to the first nonzero.

Reply:
Right, so we just want to start with the first day that has a non-zero value. 

Case 1: 
all values are non-zero, start with the first entry

Case 2:
there are values that are zero, start with the first entry that is non-zero

Does this answer your question?

Reply:
I think case 1 is actually "all values are zero" though (because `len(nonzeros) == 0` means there are only zeros). 😁

Reply:
So, if all entries are zero - the data isn't providing any information, and it should not affect future predictions. I don't anticipate this happening, but we could code up a different way of dealing with that - but I don't think it is too worthwhile

Reply:
Yeah, I agree 100%.  I just wanted to confirm my understanding.  No need to spend more time on it.  Thanks!